00-Install_and_Setup/README.md

```diff
@@ -95,19 +103,11 @@ The python version and all needed packages, including astropy, are listed in the
 
 On Mac or Linux, open your terminal.
```

Comment:
Still have an OS specific reference here.

Reply:
Better to say, "Open a terminal"? Although I guess on Windows that is within WSL?